src/ainft.ts

```diff
@@ -22,13 +22,19 @@ import {
   AIN_BLOCKCHAIN_CHAIN_ID,
   AIN_BLOCKCHAIN_ENDPOINT,
 } from './constants';
-import { setEnv } from './utils/env';
+
+export interface ClientOptions {
+  privateKey?: string;
+  signer?: Signer;
+  baseURL?: string | null;
+  blockchainURL?: string | null;
```

Comment:
minor) how about 'blockchainUrl' here and other places?

Reply:
done!